src/index.ts

```diff
@@ -225,6 +225,14 @@ export class PartitionConsumer {
         await this.inner.stream(offset, event.emit.bind(event))
         return
     }
+
+    async getIterator(offset: Offset): Promise<any> {
```

Comment:
This should really return a `Promise<AsyncIterator<string>>` but the ts compiler complains when I do it. Thoughts?